youtube_dl/extractor/gamestar.py

```diff
@@ -9,21 +9,37 @@
 
 
 class GameStarIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?gamestar\.de/videos/.*,(?P<id>[0-9]+)\.html'
-    _TEST = {
-        'url': 'http://www.gamestar.de/videos/trailer,3/hobbit-3-die-schlacht-der-fuenf-heere,76110.html',
-        'md5': '96974ecbb7fd8d0d20fca5a00810cea7',
-        'info_dict': {
-            'id': '76110',
-            'ext': 'mp4',
-            'title': 'Hobbit 3: Die Schlacht der Fünf Heere - Teaser-Trailer zum dritten Teil',
-            'description': 'Der Teaser-Trailer zu Hobbit 3: Die Schlacht der Fünf Heere zeigt einige Szenen aus dem dritten Teil der Saga und kündigt den...',
-            'thumbnail': r're:^https?://.*\.jpg$',
-            'timestamp': 1406542020,
-            'upload_date': '20140728',
-            'duration': 17
-        }
-    }
+    _VALID_URL = r'https?://(?:www\.)?game(?:pro|star)\.de/videos/.*,(?P<id>[0-9]+)\.html'
+    _TESTS = [
+        {
+            'url': 'http://www.gamestar.de/videos/trailer,3/hobbit-3-die-schlacht-der-fuenf-heere,76110.html',
+            'md5': 'ee782f1f8050448c95c5cacd63bc851c',
+            'info_dict': {
+                'id': '76110',
+                'ext': 'mp4',
+                'title': 'Hobbit 3: Die Schlacht der Fünf Heere - Teaser-Trailer zum dritten Teil',
+                'description': 'Der Teaser-Trailer zu Hobbit 3: Die Schlacht der Fünf Heere zeigt einige Szenen aus dem dritten Teil der Saga und kündigt den...',
+                'thumbnail': r're:^https?://.*\.jpg$',
+                'timestamp': 1406542380,
+                'upload_date': '20140728',
+                'duration': 17,
+            }
+        },
+        {
+            'url': 'http://www.gamepro.de/videos/top-10-indie-spiele-fuer-nintendo-switch-video-tolle-nindies-games-zum-download,95316.html',
+            'md5': '94e8fd02da27b01989cbb2786e766e4b',
+            'info_dict': {
+                'id': '95316',
+                'ext': 'mp4',
+                'title': 'Top 10: Indie-Spiele für Nintendo Switch - Video: Tolle Nindies-Games zum Download',
+                'description': 'Wer behauptet, im letzten Jahr seien nicht ausreichend Spiele für die Switch\xa0herausgekommen, der hat offensichtlich nicht richtig...',
+                'thumbnail': r're:^https?://.*\.jpg$',
+                'timestamp': 1515173400,
+                'upload_date': '20180105',
+                'duration': 615,
+            }
```

Comment:
Make only_matching.

Reply:
I've removed everything except the URL and added the only_matching tag. Is that as desired?